framework/src/PhaseSpaceBoundary.cpp

```diff
@@ -587,7 +587,7 @@ unsigned int PhaseSpaceBoundary::GetDiscreteIndex( DataPoint* Input, const bool
 		cerr << "This is a SERIOUS MISCONFIGURATION!!! Exiting :(" << endl;
 		Input->Print();
 		this->Print();
-		DebugClass::SegFault();
```

Comment:
I originally got rid of the function entirely because of compiler warnings, and I wanted to compile with `-Werror`
```
framework/src/DebugClass.cpp: In static member function ‘static void DebugClass::SegFault()’:
framework/src/DebugClass.cpp:70:1: warning: ‘noreturn’ function does return
```

This is caused by the block of code [here](https://github.com/gcowan/RapidFit/blob/master/framework/include/DebugClass.h#L32) in `DebugClass.h`.

Why not just do `raise(SIGSEGV)`?